wire-ios/Wire-iOS/Resources/es.lproj/Localizable.strings

```diff
@@ -1958,3 +1961,23 @@
 "credentials.general-error.alert.title" = "Invalid Information";
 "credentials.general-error.alert.message" = "These account credentials are incorrect. Please verify your details and try again.";
 "add-email-password-step.cta-button.title" = "Confirmar";
+
+// MARK: - Sketch Colors
+"drawing.colors.black" = "black";
```

Comment:
I am sure we can get these translated as it's basic colors, is the way to go to accept suggestions on crowding  ?